eq-author/src/App/QuestionnaireDesignPage/index.test.js

```diff
@@ -160,6 +161,78 @@ describe("QuestionnaireDesignPage", () => {
     });
   });
 
+  describe("onAddCalculatedSummaryPage", () => {
+    it("should add new page below current page", () => {
+      wrapper.find(NavigationSidebar).simulate("addCalculatedSummaryPage");
+
+      expect(mockHandlers.onAddCalculatedSummaryPage).toHaveBeenCalledWith(
+        section.id,
+        page.position + 1
+      );
+    });
+
+    it("should add page at start of section if page not found", () => {
+      wrapper.setProps({
+        questionnaire: {
+          ...questionnaire,
+          sections: [
+            {
+              ...section,
+              pages: [],
+            },
+          ],
+        },
+      });
+
+      wrapper.find(NavigationSidebar).simulate("addQuestionPage");
+
+      wrapper.find(NavigationSidebar).simulate("addCalculatedSummaryPage");
+
+      expect(mockHandlers.onAddCalculatedSummaryPage).toHaveBeenCalledWith(
+        section.id,
+        1
```

Comment:
This isn't the start of the section as described in the test title so I'm not sure what this is proving

Reply:
Good question 🤷‍♂️